server/s3mock.js

```diff
@@ -1,55 +1,23 @@
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0012.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0013.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0014.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0015.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0016.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0017.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0018.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0019.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0020.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0021.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0022.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0023.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0024.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0025.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0026.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0027.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0028.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0029.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0030.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0031.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0032.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0033.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0034.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0035.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0036.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0037.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0038.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0039.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0040.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0041.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0042.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0043.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0044.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0045.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0046.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0047.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0048.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0049.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0050.jpg`,
-  `https://fecimagemodal.s3-us-west-2.amazonaws.com/images/0051.jpg`
-];
-
-module.exports = mockImages;
+const generateMockData = function () {
+  const mockURLs = [];
+  for (let i = 1; i < 52; i += 1) {
+    if (i < 10) {
```

Comment:
You can avoid these conditionals if you simply pad the id to be four characters long. Look up the `padStart` function.

Reply:
I was completely unaware of this. Thanks for the read. Refactoring accordingly